tests/ui/patrons/test_patrons_api.py

```diff
@@ -47,7 +45,7 @@ def test_patron_create(app, roles, lib_martigny, librarian_martigny_data_tmp,
 
     wrong_librarian_martigny_data_tmp = deepcopy(librarian_martigny_data_tmp)
     wrong_librarian_martigny_data_tmp.pop('first_name')
-    with pytest.raises(ValidationError):
+    with pytest.raises(Exception):
```

Comment:
Same here. If an Exception is raised during the `create` call (during validation process). It should be catched and re-raise a `ValidationError`